exercicioLogica16/src/main/java/org/example/Emprestimo.java

```diff
@@ -1,43 +1,71 @@
 package org.example;
 
 
+import java.util.ArrayList;
+
 public class Emprestimo {
     private Livro livro;
 
     private Pessoa pessoa;
 
-    public Emprestimo(Livro livro, Pessoa pessoa) {
-        this.livro = livro;
-        this.pessoa = pessoa;
-    }
+    private ArrayList<String> membros = new ArrayList<>();
+    private ArrayList<Integer> docBiblioteca = new ArrayList<>();
+
+    private ArrayList<String> listaLivros = new ArrayList<>();
 
 
-    public void pegarLivroEmprestado(String titulo, String autor, String nome, int documentoBiblioteca){
-        pessoa.getPessoa().add(nome);
-        pessoa.getDocBiblioteca().add(documentoBiblioteca);
 
-        livro.getListaLivros().add(titulo + " " + autor);
+    public void pegarLivroEmprestado(Livro livro, Pessoa pessoa){
+        membros.add(pessoa.getNome());
```

Comment:
a pessoa do emprestimo poderia receber a pessoa sendo enviada por parametro no construtor

`this.pessoa = pessoa;`